velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -68,20 +68,21 @@ void WindowFuzzer::addWindowFunctionSignatures(
   }
 }
 
-std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
+std::string WindowFuzzer::generateFrameClause(const uint32_t& typeOfFrame) {
   auto frameType = [](int value) -> const std::string {
     switch (value) {
       case 0:
-        return "RANGE";
-      case 1:
         return "ROWS";
+      case 1:
+        return "RANGE";
+      // K-RANGE frame
```

Comment:
The separate case can be removed here. Alternatively a boolean variable could be passed which indicates whether k-range frames could be generated for the given input.

Reply:
@pramodsatya I wanted to have case 2 here to represent K-Range frames so I don't have to pass another argument into this generateFrameClause() function

Reply:
Removed case 2 since it's handled by bool isSingleSortingKey passed in to function now